File: Platform/codesignal/Intro/19.areEquallyStrong/solution.go
package main

import (
	"fmt"
	"math"
)

func solution(yl int, yr int, fl int, fr int) bool {
	return math.Max(float64(yl), float64(yr)) == math.Max(float64(fl), float64(fr)) && (yl+yr) == (fl+fr)
}

func main() {
	yl, yr, fl, fr := 10, 15, 15, 10
	fmt.Print(solution(yl, yr, fl, fr))
}

File: Platform/codesignal/Intro/19.areEquallyStrong/solution.py
def solution(yl, yr, fl, fr):
  return max(yl, yr) == max(fl, fr) and (yl+yr) == (fl+fr)

def solution(yourLeft, yourRight, friendsLeft, friendsRight):
    return sorted([yourLeft,yourRight])==sorted([friendsLeft,friendsRight])

print(solution(10,15,15,10))